src/js/background.js

```diff
@@ -18,3 +18,46 @@ chrome.extension.onConnect.addListener(function(port) {
     });
 });
 
+/**
+ * Holds the state of debug mode. True means we should debug the page.
+ */
+let debug = false;
+let hasAPIs = chrome.webRequest;
+
+/**
+ * Responds to clicks on the extension's icon. Toggles debug mode.
+ */
+chrome.browserAction.onClicked.addListener((tab) => {
+
+    debug = !debug;
+    chrome.browserAction.setTitle({
+        title: debug ? 'Sidecar Debug: ON' : 'Sidecar Debug: OFF'
+    });
+    // chrome.browserAction.setIcon({
+    //     path: debug ? 'icon-on.png' : 'icon-off.png'
+    // });
```

Comment:
this is commented because I really think we should have a grayed icon, but I didn't have time to create one yet.
I think we can also improve the one we have by creating a png from a SVG one (and have the size that we want).

I'll work on that tomorrow